### ISSUE TITLE
bad regex in flables.yml

### ISSUE THREAD

**Issue by jalphonso** [Juniper/healthbot-rules#455]
https://github.com/Juniper/healthbot-rules/blob/master/juniper_official/Solutions/EVPN-VXLAN/flables.yml

In the original rule the .* eats all the leading digits leaving only the last digit

      token-count: 'Combined global token count.*(\d+)'
      token-words-count: 'Combined global token words count.*(\d+)'

Device truncated output

```
net.prds_token_cnts:
Combined global token count: 109
Combined global token words count: 285
```

Healthbot was only showing the last digit. With the below change, all digits are saved properly

      token-count: 'Combined global token count: (\d+)'
      token-words-count: 'Combined global token words count: (\d+)'

**Comment by gvarao** [comment]
Hi Joseph,

Thanks for finding issue and solution.

Gowri,

Please change regex as Joesph suggested and create pull request.

Regards,
Venkat(GV)

**Comment by gvarao** [comment]
Sure GV.

Thanks,
Gowrisankar

From: venkata Gajjarapu <gvenkata@juniper.net>
Date: Monday, 7 December 2020 at 9:15 AM
To: Juniper/healthbot-rules <reply@reply.github.com>, Juniper/healthbot-rules <healthbot-rules@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>, Gowrisankar Seerangasamy Valayapalayam <sankarvs@juniper.net>
Subject: Re: [Juniper/healthbot-rules] bad regex in flables.yml (#455)

Hi Joseph,

Thanks for finding issue and solution.

Gowri,

Please change regex as Joesph suggested and create pull request.

Regards,
Venkat(GV)

**Comment by gvarao** [comment]
Fixed with #457

**Comment by gvarao** [comment]
Code merged in development branch. Please check and revert
